metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -152,7 +153,11 @@
                    [conn db]
                    (let [conditions [`(>= :id ~start-index)
                                      `(< :id ~(+ start-index batch-size))]
-                         _ (info "Finding batch for provider" provider-id "from id >=" start-index " and id <" (+ start-index batch-size))
+                         _ (info (format "Finding batch for provider [%s] concept type [%s] from id >= %s and id < %s"
```

Comment:
@littleidiot40, Can you review this logging change? I believe it gives you better info on the batch processing. You may need to  update your script if it doesn't work with the new logging.